Source/items.cpp

```diff
@@ -1571,7 +1565,9 @@ void SetupBaseItem(Point position, _item_indexes idx, bool onlygood, bool sendms
 	GetSuperItemSpace(position, ii);
 	int curlv = ItemsGetCurrlevel();
 
-	SetupAllItems(*MyPlayer, item, idx, AdvanceRndSeed(), 2 * curlv, 1, onlygood, false, delta);
+	SetupAllItems(*MyPlayer, item, idx, AdvanceRndSeed(), 2 * curlv, 1, onlygood, delta);
+	SetupItem(item);
+	TryRandomUniqueItem(item, idx, 2 * curlv, 1, onlygood, delta);

```

Comment:
Is this an error? Most of these call `SetupItem()` after `TryRandomUniqueItem()`.

Reply:
Shouldn't make a difference, but changing it for consistency.

Reply:
If it makes no difference, then why was the call to `SetupItem()` removed from `SetupAllItems()`?